task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,9 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if ((num%3 == 0) && (num%5 == 0)) return 'FizzBuzz';
+    if (num%3 == 0) return 'Fizz';
+    return (num%5 == 0) ? 'Buzz' : num;
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"